test/end-to-end/Exception Handling/exception handling.py

```diff
@@ -0,0 +1,11 @@
+-
+try:
+    # ...
+
```

Comment:
Is this blank line standard per Python style guidelines? I thought it wasn't, but am not very familiar.